.github/workflows/build.yml

```
@@ -19,6 +20,7 @@ on:
 
 jobs:
   setup:
+    if: ${{ github.repository == 'liquibase/liquibase' || (github.repository != 'liquibase/liquibase' && github.event.label.name == 'SafeToBuild') }}
```

You could simplify this to: 
```suggestion
    if: ${{ github.repository == 'liquibase/liquibase' || github.event.label.name == 'SafeToBuild' }}
```

---

It's not `github.event.label.name` that matters, because we want it to build on all events as long as at some point it was labeled.

So it probably needs to be some sort of "does the PR have the SaveToBuild label on it" which is true when it's first labeled and also on every future build.